### ISSUE TITLE
Не работает ответ на комментарий

### ISSUE THREAD

**Issue by satantx** [campusboy87/extend-comment#1]
Добрый день. Не работает проверка на "ответ на комментарий". Хоть рейтинг скрыт, он всё равно его проверяет и выдаёт ошибку. Подскажите как исправить. Спасибо!

**Comment by satantx** [comment]
Вроде сделал сам) Надо заменить строчку тут
```
// // Проверяем, заполнено ли поле "Рейтинг"
...
if ( isset( $_REQUEST['comment_parent'] ) && $_REQUEST['comment_parent'] > 0 ) {
return $commentdata;
}
```

**Comment by campusboy87** [comment]
Здравствуйте. А вы брали код отсюда? Просто я этот момент исправлял [больше месяца назад](https://github.com/campusboy87/extend-comment/commit/62cc5cf502a54dbb74481d688a8b56562d4c1937).

**Comment by satantx** [comment]
Да, и он не работал из за `$_REQUEST['action']` такого значения форма вообще не передаёт. Может я не прав, поправьте)

**Comment by campusboy87** [comment]
Ну форму может тема менять как угодно, не знаю как подгадать под всех :-) модификацию я писал на основе данных о баге от одного из читателей сайта. Этот Issue не закрываю, погляжу Ваш вариант, может быть он более универсальным окажется.

**Comment by satantx** [comment]
> модификацию я писал на основе данных о баге от одного из читателей сайта

Понял) а я подумал универсальный метод! Всё равно спасибо за плагин!
